runtime/doc/starting.txt

```diff
@@ -545,13 +545,13 @@ accordingly.  Vim proceeds in this order:
 	":version" command.  Mostly it's "$VIM/vimrc".
 	For the Macintosh the $VIMRUNTIME/macmap.vim is read.
 
-	  *VIMINIT* *.vimrc* *_vimrc* *EXINIT* *.exrc* *_exrc* *$MYVIMRC*
+	  *NVIMINIT* *.vimrc* *_vimrc* *EXINIT* *.exrc* *_exrc* *$MYVIMRC*
```

Comment:
Ok, then probably `.nvimrc` tag should be added instead, and `nvimrc` tag after `vimrc` above  as `:help nvimrc` fails right now, but yes this unrelated to this PR.